backend/bot/parseMatchResultPrefixSuffixConfig.js

```diff
@@ -0,0 +1,16 @@
+const parseConfigPart = configPart => {
+  const splitConfigPart = configPart.split(',')
+  if (splitConfigPart.length !== 4) {
+    throw Error(`Could not parse config part "${splitConfigPart}"`)
+  }
+  return ({
```

Comment:
1. wondering, why the extra parentheses in return ({}) ? and not just return {} ? It it's a normal JS style, keep it like it is ;)
2. i'd probably need to mull this over, but it seems that this function is tightly coupled with the match reporter, so would consider putting it in the same file. alternatively would at least consider renaming this loong funciton name to parseMatchReportDecorables(...) or somethign such

Reply:
1. http://jamesknelson.com/javascript-return-parenthesis/
2. Gonna put it within the match reporter.